### ISSUE TITLE
[Feature] Add basic `.editorconfig` file while creating new Clarinet project

### ISSUE THREAD

**Issue by LNow** [hirosystems/clarinet#279]
Clarity contracts must be saved with Unix line endings and on Windows machines some users have to manually configure their IDE to enforce it.

Maybe to make it a bit easier we should add to the list of suggested plugins: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig, and also add basic `.editorconfig` file during new project creation?

`.editorconfig` could look like this:
```
[*.{ts,clar,json,toml}]
end_of_line = lf
trim_trailing_whitespace = true
```

**Comment by unclemantis** [comment]
I am using .vscode/settings.json

Anyone else?

**Comment by obycode** [comment]
Yes, and we are already generating .vscode/settings.json to enable Deno. I think we could solve this by adding the following to the generated settings.json:

```json
    "files.eol": "\n",
```

**Comment by LNow** [comment]
`.vscode/settings.json` is VS Code specific.
`.editorconfig` is more agnostic (https://editorconfig.org/#pre-installed)

**Comment by unclemantis** [comment]
> Yes, and we are already generating .vscode/settings.json to enable Deno. I think we could solve this by adding the following to the generated settings.json:
> 
> ```json
>     "files.eol": "\n",
> ```

I make the change in Workspace for EOL. I also make it in User too.

**Comment by lgalabru** [comment]
Addressed in v0.28.0
